CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # generated by giantswarm/github actions - changes will be overwritten
```

Comment:
This must be done through our central setup

Reply:
https://github.com/giantswarm/github/pull/2656